tests/decorators/decorators.go

```diff
@@ -24,4 +24,7 @@ var (
 	NonRoot              = []interface{}{Label("verify-non-root")}
 	NativeSsh            = []interface{}{Label("native-ssh")}
 	ExcludeNativeSsh     = []interface{}{Label("exclude-native-ssh")}
+	Reenlightenment      = []interface{}{Label("Reenlightenment")}
```

Comment:
Please also check `tests/sysprep_test.go` and `tests/windows_test.go`, they have VMIs with reenlightenment as well

Reply:
Hey Thanks for reviewing,
The VM in https://github.com/kubevirt/kubevirt/blob/main/tests/windows_test.go#L250 doesn't use reenlightenment it uses other hyperv features that we should mark as well in follow up PRs :)

Reply:
Yup, you're right, my mistake :)